Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -3,10 +3,9 @@
 class Keg
   def relocate_dynamic_linkage(relocation)
     # Patching the dynamic linker of glibc breaks it.
-    return if name == "glibc"
-
     # Patching patchelf using itself fails with "Text file busy" or SIGBUS.
-    return if name == "patchelf"
+    # Patching portable-ruby causes brew tests to segfault.
+    return if ["glibc", "patchelf", "portable-ruby"].include?(name)
```

Comment:
@sjackman Is it expected that these formulae will still pour `patchelf` but not use it?

Reply:
Yeah. Pouring `patchelf` is unnecessary, since it's not used, but probably not worth the trouble of avoiding pouring it.